lib/Sema/ConstraintSystem.h

```diff
@@ -3487,6 +3487,52 @@ class OverloadSetCounter : public ASTWalker {
     return { true, expr };
   }
 };
+
+
+/// \brief Matching an array of input to an array of parameter.
```

Comment:
Alternative phrasing - Matches array of function parameters to candidate arguments.